core/src/main/java/nl/nn/adapterframework/processors/CorePipeProcessor.java

```diff
@@ -32,4 +33,9 @@ public PipeRunResult processPipe(PipeLine pipeLine, IPipe pipe, Message message,
 		return pipe.doPipe(message, pipeLineSession);
 	}
 
+	@Override
+	public PipeRunResult validate(PipeLine pipeLine, IValidator validator, Message message, PipeLineSession pipeLineSession, String messageRoot) throws PipeRunException {
+		return validator.validate(message, pipeLineSession, messageRoot);
+	}
+
```

Comment:
Ik denk dat dit binnen de scope van de pipeline processor moet blijven. Nu kan elke pipe dit en dat moeten we ook niet willen volgens mij...

Reply:
Dat willen we wel voor Pipes die een InputValidator en/of OuputValidator hebben.